experimental/DoubleAndHyperComplexes/src/Morphisms/simplified_complexes.jl

```diff
@@ -398,9 +447,6 @@ If `F` is a free resolution of either a positively graded module `M`, or a modul
 
 If `M` is not (positively) graded or its `base_ring` is not local, use `simplify` to obtain an ''improved'' resolution.
 
-!!! note
-    If `F` is not complete, the minimal free resolution is computed only up to the second last known non-zero module in the resolution `F`. 
-
```

Comment:
@HechtiDerLachs Why is this removed?

Reply:
That was for the intermediate state which we had. I reverted this change.